proto/money/folio/folio_state.proto

```diff
@@ -8,14 +8,17 @@ import "primitive/monetary_amount.proto";
 import "money/folio/posted_card_authorization.proto";
 import "money/folio/posted_charge_credit.proto";
 import "money/folio/posted_payment.proto";
+import "money/folio/posted_refund.proto";
 
 message FolioState {
 	repeated PostedChargeCredit charge_credits = 1;
 	repeated PostedCardAuthorization card_authorizations = 2;
 	repeated PostedPayment payments = 3;
+	repeated PostedRefund refunds = 4;
```

Comment:
Yep, I understand. I guess since we aren't dealing with live proto objects of this type floating around, it makes sense to insert into logical locations.